packages/harpguru-core/src/components/option-stack/utils/get-styles/get-styles.ts

```diff
@@ -0,0 +1,70 @@
+import type { TextStyle, ViewStyle } from 'react-native'
+import { Dimensions, StyleSheet } from 'react-native'
+
+import { colors, getSizes } from '../../../../styles'
+
+type OptionStackStyles = {
+  titleFontSize: number
+  titleSection: ViewStyle
+  titleWrapper: ViewStyle
+  titleText: TextStyle
+  subTitleText: TextStyle
+  listSection: ViewStyle
+  listWrapper: ViewStyle
+  optionText: TextStyle
+}
+
+export const getStyles = (): OptionStackStyles => {
+  const sizes = getSizes()
+  const { ['9']: titleFontSize } = sizes
+  const { ['11']: gutterSize } = sizes
+  // TODO: turn this in to a util since it's used elsewhere
```

Comment:
TODO here

Reply:
And call it `longEdge` and `shortEdge` so I'm not confused by the names any more.

Reply:
Done